library/billing/src/BillingUtilities.php

```diff
@@ -1279,17 +1279,17 @@ class BillingUtilities
 
     public static function getBillingById($id, $cols = "*")
```

Comment:
This function can be completely removed since not used anywhere in OpenEMR's codebase :)